src/include/utils/rel.h

```diff
@@ -674,10 +677,16 @@ typedef struct ViewOptions
  * 
  * This is similar to RelationIsLogicallyLogged but we only want to generate the
  * extra information for logical replication without actually WAL-logging it.
+ * Only a subset of the conditions in RelationNeedsWAL is used here because we
+ * still want this to be valid for remote relations.
+ * 
+ * Remotexact
```

Comment:
Did you mean to add some additional comment here?

Reply:
No that was intentional. Since we're making more changes to postgres now, I'm putting the Remotexact tags everywhere so that we can search for the changes later.